src/main/tv/codely/scala_intro_examples/lesson_10_enums_vs_sealed_structures/exercise_solutions/daniel_albarral/GenericPicoteriaProduct.scala

```diff
@@ -0,0 +1,18 @@
+package tv.codely.scala_intro_examples.lesson_10_enums_vs_sealed_structures.exercise_solutions.daniel_albarral
+
+import java.util.UUID
+
+trait PicoteriaProduct {
+  val id: UUID
+}
+
+trait GenericFood extends Ordered[GenericFood] {
+
+  override def compare(that: GenericFood): Int = that.getForcefulness - getForcefulness
+
+  val getForcefulness: Int
```

Comment:
It isn't idiomatic for Scala code to specify the `get` prefix as we would do in Java code. Here you have more information about it: https://docs.scala-lang.org/style/naming-conventions.html#accessorsmutators